### DISCUSSION TITLE
Custom 4xx 5xx pages

### DISCUSSION THREAD

**Discussion by LorenzoSapora** [laravel/nova-issues#4964] [Q&A] [answered]
In Nova 3 and below, we could create a nova component to replace the error pages, by creating a cue component to hijack it, but now Nova 4's codebase is obfuscated, that method no longer works.

I don't mind making a component for this, like I did in v1.0 to v3.0, but would either need pointers, or it would be nice to have this publishable or customisable within Nova.

**Comment by crynobone** [comment, marked as answer]
You should be able to override `CustomError403`, `CustomError404` and `CustomAppError` components.
